Jenkinsfile

```
@@ -3,7 +3,7 @@
 pipeline {
     agent {
         docker {
-            image 'ruby:2.4-alpine'
+            image 'ruby:3.1.3-alpine'
```

```suggestion
            image 'ruby:3.1.3'
```

The size difference between both images is not worth the problems caused by missing tools such as `bash` (for this case where we do not use a custom image)

---

Good point!